src/collapse/perf.rs

```diff
@@ -65,6 +64,9 @@ pub struct Folder {
     /// Function entries on the stack in this entry thus far.
     stack: VecDeque<String>,
 
+    /// Keep track of inlined Java functions so they can be annotated with "_[i]".
+    java_inline: Vec<String>,
```

Comment:
I'd like to call this `cache_line` or something, just because it doesn't specifically _need_ to be Java inlining (although in practice I suppose it basically always is).